src/app/home/pages/ClustersPage/components/AddEditClusterModal/AddEditClusterForm.tsx

```diff
@@ -181,12 +182,13 @@ const InnerAddEditClusterForm = (props: IOtherProps & FormikProps<IFormValues>)
         {/*
           // @ts-ignore issue: https://github.com/konveyor/mig-ui/issues/747 */}
         <TextInput
+          data-testid="account-token"
```

Comment:
Sorry I missed this before, but since we're not using this data-testid anymore can you remove it?